src/Hintman/Config.hs

```diff
@@ -44,9 +44,14 @@ hintmanConfiguationCodec = HintmanConfig
 
 loadFileConfig :: FilePath -> IO HintmanConfig
 loadFileConfig path =
-    Toml.decodeFile hintmanConfiguationCodec path `catch` catchLoad
+    (Toml.decodeFile hintmanConfiguationCodec path `catch` catchExcAndReturnDefaultConfig)
+      `catch` catchExcAndReturnDefaultConfig
   where
-    catchLoad :: LoadTomlException -> IO HintmanConfig
-    catchLoad _ = do
-        print $ "Hintman failed to load config from " <> path
-        pure defaultHintmanConfig
+    catchExcAndReturnDefaultConfig :: SomeException -> IO HintmanConfig
+    catchExcAndReturnDefaultConfig e = case e of
+        Exc (_ :: IOException) -> do
+            print $ "Config '" <> path <> "' does not exist! Using default configuration"
+            pure defaultHintmanConfig
+        _ -> do
```

Comment:
Also, we know that this is `TomlException`. For more pleasant experience working on the project, we probably want to provide the exception description in here, not just "Failed to load". What do you think?

Reply:
Right, that makes sense. Do you have any suggestion on what the catch all clause here should be? Maybe a generic error message with a `show err`?

Reply:
Yes! I think this will work okay here 👍

Reply:
OK, I'll work on an updated PR. Thanks @vrom911!